iaso/tests/fixtures/odk_invalid_xlsform_expected_errors.json

```
@@ -1006,18 +1006,18 @@
         "severity": "error"
     },
     {
-        "message": "choices not for 'nonexistingchoices' and list_name 'doenst_exist'",
+        "message": "choices not found for 'nonexistingchoices' and list_name 'doenst_exist'",
```

I think it's a typo?

```suggestion
        "message": "choices not found for 'nonexistingchoices' and list_name 'doesnt_exist'",
```

---

fixed the xlsform in fixture